docs/requirements.txt

```diff
@@ -1,6 +1,7 @@
 attrs >= 20.1.0
-furo ~= 2021.10.09
+furo >= 2024.01.29
```

Comment:
I'm not sure this change will work -- let's keep an eye on this. I remember pinning furo because of some weird dependency failing in more recent builds. Perhaps they figured out.

Reply:
I built it locally.